requestmatch/matcher.go

```diff
@@ -114,6 +115,24 @@ func compileRxs(exps []string) ([]*regexp.Regexp, error) {
 	return rxs, nil
 }
 
+func keyValToLower(kv map[string]string) map[string]string {
```

Comment:
I don't think go argues, that they are not good or useful. They only haven't found a satisfying way to add them to the lang, right?